libopae/src/mmio.c

```diff
@@ -166,7 +166,7 @@ static fpga_result map_mmio_region(fpga_handle handle, uint32_t mmio_num)
 	wsid = wsid_gen();
 	if (!wsid_add(&_handle->mmio_root,
 		      wsid,
-		      (uint64_t) NULL,
+		      (uint64_t) addr,
```

Comment:
Do we also need to add the iova, or is that done somewhere else?

Reply:
Adding MMIO pointer to work space list , ivoa is not valid in mmio, this not buffer pointer

Reply:
For MMIO, that doesn't seem necessary (no need for user space or the accelerator to know that).